trade-finance-manager-ui/server/types/view-models/record-correction/create-record-correction-request-errors-view-model.ts

```diff
@@ -0,0 +1,7 @@
+import { ErrorSummaryViewModel } from '../error-summary-view-model';
+
+export type CreateRecordCorrectionRequestErrorsViewModel = {
```

Comment:
This is only used in the CreateRecordCorrectionRequestViewModel, so please could we just pop it in that file for simplicity? Same thing for the form values type. We do things differently for the payment form values and errors because those types are reused in multiple view models.